lib/arbre/html/attributes.rb

```diff
@@ -6,7 +6,7 @@ class Attributes < Hash
       def to_s
         flatten_hash.compact.map do |name, value|
           "#{html_escape(name)}=\"#{html_escape(value)}\""
-        end.compact.join ' '
+        end.join ' '
```

Comment:
if the idea here is to "compact_blank`, because this at the moment will also add `foo=""` then this can be revisited